File: MODUL-6/Python/PRAK601-2310817220002-RaudatulSholehah.py
Kolom, Baris = map(int, input().split())
Nomor = list(map(int, input().split()))
Matrix = []
a = 0
for i in range (Kolom) :
    Matrix.append(Nomor[a:a+Baris])
    a = a + Baris
for Kolom in Matrix :
    for Nomor in Kolom :
        print(Nomor, end=' ')
    print()

File: MODUL-6/C/PRAK601-2310817220002-RaudatulSholehah.c
#include <stdio.h>
int main()
{
    int a, b, Matrix, Kolom, Baris;
    scanf("%i %i", &Baris, &Kolom);
    int Nomor[Baris][Kolom];
    for (a = 0; a < Baris; a++)
    {
        for (b = 0; b < Kolom; b++)
        {
            scanf("%i", &Matrix);
            Nomor[a][b] = Matrix;
        }
    }
    for (a = 0; a < Baris; a++)
    {
        for (b = 0; b < Kolom; b++)
        {
            printf("%i ", Nomor[a][b]);
        }
        printf("\n");
    }
    return 0;
}